snuba/snuba_migrations/generic_metrics/0052_counters_raw_add_sampling_weight.py

```diff
@@ -0,0 +1,57 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+
+    storage_set_key = StorageSetKey.GENERIC_METRICS_COUNTERS
+
+    local_table_name = "generic_metric_counters_raw_local"
+    dist_table_name = "generic_metric_counters_raw_dist"
+
+    columns: Sequence[tuple[Column[MigrationModifiers], str | None]] = [
+        (
+            Column(
+                "sampling_weight",
+                UInt(64, MigrationModifiers(default=str("1"))),
```

Comment:
Let's add CODEC(ZSTD(1)) encoding to the column

Reply:
Fixed!